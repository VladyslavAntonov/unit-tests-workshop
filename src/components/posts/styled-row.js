import { TableRow } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';

export const StyledRow = withStyles((theme) => ({
  root: {
    '&:nth-of-type(odd)': {
      backgroundColor: theme.palette.action.hover
    }
  }
}))(TableRow);
