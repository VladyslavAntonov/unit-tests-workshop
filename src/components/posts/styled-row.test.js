import { shallow } from 'enzyme';

import { StyledRow } from './styled-row';

describe('StyledRow', () => {
  let component;

  beforeEach(() => {
    component = shallow(<StyledRow>content</StyledRow>);
  });

  it('matches the snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});

