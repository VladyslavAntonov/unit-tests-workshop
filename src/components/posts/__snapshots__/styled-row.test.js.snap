// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledRow matches the snapshot 1`] = `
<WithStyles(ForwardRef(TableRow))
  classes={
    Object {
      "root": "WithStyles(ForwardRef(TableRow))-root-1",
    }
  }
>
  content
</WithStyles(ForwardRef(TableRow))>
`;
